classes/core/APIRouter.inc.php

```diff
@@ -77,7 +80,16 @@ function route($request) {
 		// Ensure slim library is available
 		require_once('lib/pkp/lib/vendor/autoload.php');
 
-		$sourceFile = sprintf('api/%s/%s/index.php', $this->getVersion(), $this->getEntity());
+		$sourceFile = null;
+		$entity = $this->getEntity();
+		$sourceFile = sprintf('api/%s/%s/index.php', $this->getVersion(), $entity[0]);
+
+		if (!empty($entity[1])) {
```

Comment:
swagger document would be a good reference. https://github.com/kaschioudi/ojs/blob/2163-1503-fix/docs/dev/swagger.yaml